rust/lance-table/src/rowids/bitmap.rs

```diff
@@ -0,0 +1,227 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright The Lance Authors
+
+#[derive(PartialEq, Eq, Clone)]
+pub struct Bitmap {
+    pub data: Vec<u8>,
```

Comment:
This is a good idea. There are some details with serialization to figure out, but I think it would work well and eliminate the need for this file. I'm going to leave this as a TODO for now.